cfg/config.go

```diff
@@ -319,20 +321,55 @@ func (c Config) validateConfig() error {
 		return errors.New("GitHub token required")
 	}
 
-	jUser := c.cmdConfig.GetString("jira-user")
-	if jUser == "" {
-		return errors.New("Jira username required")
-	}
+	if c.cmdConfig.GetString("jira-user") != "" || c.cmdConfig.GetString("jira-pass") != "" {
+		c.log.Debug("Using Basic Authentication")
 
-	jPass := c.cmdConfig.GetString("jira-pass")
-	if jPass == "" {
-		fmt.Print("Enter your JIRA password: ")
-		bytePass, err := terminal.ReadPassword(int(syscall.Stdin))
-		if err != nil {
-			return errors.New("Jira password required")
+		jUser := c.cmdConfig.GetString("jira-user")
+		if jUser == "" {
+			return errors.New("JIRA username required")
+		}
+
+		jPass := c.cmdConfig.GetString("jira-pass")
+		if jPass == "" {
+			fmt.Print("Enter your JIRA password: ")
```

Comment:
huh funny, `fmt.Print` is almost never used. I had to double check that it was a real func.